src/components/tree/phyloTree/renderers.js

```diff
@@ -172,7 +172,7 @@ export const drawBranches = function drawBranches() {
   if (!("branchTee" in this.groups)) {
     this.groups.branchTee = this.svg.append("g").attr("id", "branchTee");
   }
-  if (this.layout === "clock" || this.layout === "unrooted") {
+  if (this.layout === "clock" || this.layout === "unrooted" || params.isTreeSplitByTrait) {
```

Comment:
This is causing the "T" branches (the vertical part of a branch) not to be drawn, which is one reason why the subtrees aren't joining up. Drawing the T branches in this PR doesn't look good as it stands, I think because the y-values of parents and children are far apart when they should be similar, and the coordinates of the T-branch are such that it spans the y-values of the nodes children (see other comments).